test/db/archos/linux-x64/dbg_drt

```diff
@@ -135,188 +135,188 @@ CMDS=<<EOF
 drt all~[0]
 EOF
 EXPECT=<<EOF
-ymm15
-ymm14
-ymm13
-ymm12
-ymm11
-ymm10
-ymm9
-ymm8
-ymm7
-ymm6
-ymm5
-ymm4
-ymm3
-ymm2
-ymm1
-ymm0
-x64
-xmm15h
-xmm15
-xmm14l
-xmm14h
-xmm14
-xmm13l
-xmm13h
-xmm13
-xmm12l
-xmm12h
-xmm12
-xmm11l
-xmm11h
-xmm11
-xmm10l
-xmm10h
-xmm10
-xmm9l
-xmm9h
-xmm9
-xmm8l
-xmm8h
-xmm8
-xmm7l
-xmm7h
-xmm7
-xmm6l
-xmm6h
-xmm6
-xmm5l
-xmm5h
-xmm5
-xmm4l
-xmm4h
-xmm4
-xmm3l
-xmm3h
-xmm3
-xmm2l
-xmm2h
-xmm2
-xmm1l
-xmm1h
-xmm1
-xmm0l
-xmm0h
-xmm0
-mm7
-mm6
-mm5
-mm4
-mm3
-mm2
-mm1
-mm0
-st7
-st6
-st5
-st4
-st3
-st2
-st1
-st0
-mxcr_mask
-mxcsr
-frdp
-frip
-fop
-ftw
-swd
-cwd
-dr7
-dr6
-dr3
-dr2
-dr1
-dr0
-gs_base
-fs_base
-es
-ds
-gs
-fs
-ss
-orax
-of
-df
-if
-tf
-sf
-zf
-af
-pf
-cf
-eflags
-rflags
-cs
-rip
-bpl
-bp
-ebp
-rbp
-spl
-sp
-esp
-rsp
-dil
-di
-edi
-rdi
-sil
-si
-esi
-rsi
 r15b
 r15w
+cwd
 r15d
 r15
+dr0
+ymm0
+swd
+ftw
+fop
 r14b
 r14w
 r14d
 r14
+dr1
+frip
 r13b
 r13w
 r13d
 r13
+dr2
+frdp
 r12b
 r12w
 r12d
+mxcsr
 r12
+dr3
+mxcr_mask
```

Comment:
random order :?

Reply:
This corresponds to the offset of the register: dr3 is at 24 and mxcr_mask at 28:
dr3: https://github.com/radareorg/radare2/blob/master/libr/anal/p/anal_x86_cs.c#L3576
mxcr_mask: https://github.com/radareorg/radare2/blob/master/libr/anal/p/anal_x86_cs.c#L3604

Reply:
Sorry, wrong links, this is the file: https://github.com/radareorg/radare2/blob/2dfa75cc470261e45030fc4dac1fdba2c57d3dcd/libr/debug/p/native/linux/reg/linux-x64.h (but same offsets)